src/main/java/ru/hogwarts/school/controller/StudentController.java

```diff
@@ -40,12 +51,18 @@ public ResponseEntity<Student> editStudent(Student student) {
     }
 
     @DeleteMapping("{id}")
-    public void  deleteStudent(@PathVariable Long id) {
+    public void deleteStudent(@PathVariable Long id) {
         studentService.deleteStudent(id);
     }
 
     @GetMapping("/search_age/{age}")
-    public Map<Long,Student> getStudentByAgeList(@PathVariable Long age) {
+    public Map<Long, Student> getStudentByAgeList(@PathVariable Long age) {
         return studentService.findStudentsByAge(age);
     }
+
+    @GetMapping("/search_age")
```

Comment:
Путь не соответствует REST, в пути не должно быть глаголов
```suggestion
    @GetMapping(params = {"min", "max"})
```